docs/sources/operators-guide/monitor-grafana-mimir/requirements.md

```diff
@@ -20,29 +20,29 @@ If you are not, then continue reading.
+| Label name  | Configurable? | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| :---------- | :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| `cluster`   | Yes           | The Kubernetes cluster or datacenter where the Mimir cluster is running. You can configure the cluster label via the `per_cluster_label` field in the mixin configuration.                                                                                                                                                                                                                                                                                                                                                              |
+| `namespace` | Yes           | The Kubernetes namespace where the Mimir cluster is running. You can configure the namespace label via the `per_namespace_label` field in the mixin configuration.                                                                                                                                                                                                                                                                                                                                                                      |     |
+| `job`       | Yes           | A prefix (by default `<namespace>/`) followed by the Mimir component. When running in monolithic mode, the `<component>` should be `mimir`. When running in microservices mode, the `<component>` should be the name of the specific Mimir component (singular), like `distributor`, `ingester` or `store-gateway`. In the mixin configuration; you can configure the prefix via the `job_prefix` field, the label name via the `per_job_label` field, and the regular expressions used to match components with the `job_names` field. |
```

Comment:
```suggestion
| `job`       | Yes           | A prefix (by default `<namespace>/`) followed by the Mimir component. When running in monolithic mode, make sure that the `<component>` is `mimir`. When running in microservices mode, make sure that the `<component>` is the name of the specific Mimir component (singular), such as `distributor`, `ingester`, or `store-gateway`. In the mixin configuration, you can configure the prefix via the `job_prefix` field, the label name via the `per_job_label` field, and the regular expressions that are used to match components via the `job_names` field. |
```

Reply:
Rationale: should is generally ambiguous. Does a user need to do something or is doing that ideal and not required.